package.json

```diff
@@ -29,5 +29,8 @@
   },
   "scripts": {
     "test": "ava test/test.js && eslint *.js test/*.js"
-  }
+  },
+  "files": [
+    "index.js"
```

Comment:
do we need to add the LICENSE file too?

Reply:
`package.json`, `LICENSE`, and `README.md` is by default and `files` do not affect them